python/app/g2panel

```diff
@@ -176,22 +180,76 @@ class MainWindow(QtWidgets.QMainWindow):
             # filter=file_filter,
             # initialFilter='Excel File (*.xlsx *.xls)'
         )
-        self.zmq_rcv.load_mask(response[0])
+        if (response[0]):
+            self.zmq_rcv.load_mask(response[0])
 
 
 
     #updating values from detector to GUI
     def update_fields(self):
         """Update all fields in the GUI with current values from the detector"""
-        self.frameSpinBox.setValue(self.det.frames)
-        self.nameLineEdit.setText(self.rcv.fname)
-        self.pathLineEdit.setText(self.rcv.fpath)
-        self.exposureSpinBox.setValue(self.det.exptime)
-        self.periodSpinBox.setValue(self.det.period)
         self.triggerSpinBox.setValue(self.det.triggers)
-        self.delayTriggerSpinBox.setValue(self.det.delay)
         self.indexSpinBox.setValue(self.rcv.findex)
         self.highVoltageSpinBox.setValue(self.det.highvoltage)
+        self.frameSpinBox.setValue(self.det.frames)
+        self.nameLineEdit.setText(self.rcv.fname)
+        self.pathLineEdit.setText(self.rcv.fpath)
+        
+        #converting to righ unit of time
+        tExp = (self.det.exptime)
+        #TODO remove this line after fixing bug
+        #to check the value
+        print({tExp})
+        #Checking if the time is in nano seconds as python uses seconds
+        if tExp < 100e-9:
```

Comment:
I think later it would be good to put these checks in a separate function, but for now it's ok